src/types.ts

```diff
@@ -76,7 +76,7 @@ export interface EventFilterOptions<T> extends EventFilter<T> {
     name?: string;
 }
 
-export enum SolidoProviderType {
+export enum CementoProviderType {
```

Comment:
src/types.ts look for this file for the handling of the variables you should look for more generic names for:
    Connex = ‘connex’,
    Thorify = ‘thorify’,
    Web3 = ‘web3’,
Because those are the ones that were used when creating solido in decent-bet, but now that cemento will be something more abstract, you should look for more generic names I think that those parameters will vary by blockchain !!, therefore you must analyze how to handle it in a way of abstract it from the context of the blockchain to use because those names are for VeChain, but in Ethereum they would be others and in Binance Smart Chain others and so on !!